src/AaveV3StrategicAssets_20230606/DeployStrategicAssetsManager.s.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {EthereumScript} from 'aave-helpers/ScriptUtils.sol';
+
+import {StrategicAssetsManagerPayload} from './StrategicAssetsManagerPayload.sol';
+
+contract DeployAssetManagementContracts is EthereumScript {
+  function run() external broadcast {
+    new StrategicAssetsManagerPayload();
+  }
+}
+
+contract DeployStrategicAssetsProposal is EthereumScript {
```

Comment:
@efecarranza I think we don't need a proposal for this no, as we can just deploy and set the permission of the asset manager to the executor and guardian and that's it... or am I missing something.